ConfigSpace/forbidden.py

```diff
@@ -89,7 +92,7 @@ def __init__(self, hyperparameter, value):
                              "'%s'" % (hyperparameter, str(value)))
         self.value = value
 
-    def is_forbidden(self, instantiated_hyperparameters, strict=True):
+    def is_forbidden(self, instantiated_hyperparameters: Hyperparameter, strict: bool=True) -> bool:
```

Comment:
Please check the code by running it, but I think that `instantiated_hyperparameters` is a `dict`. Please also annotate `_is_forbidden`.